Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -583,24 +583,41 @@ public static boolean isCaseOpen() {
     }
 
     /**
+     * Deprecated. Use getOpenCase() instead.
+     * 
```

Comment:
This is not the correct way to comment on deprecation. Please use the @deprecated comment instead. There should be an example of this for other deprecated Case methods in this file.